android/src/main/kotlin/com/eopeter/flutter_mapbox_navigation/models/SimpleWaypoint.kt

```diff
@@ -0,0 +1,18 @@
+package com.eopeter.flutter_mapbox_navigation.models
+
+import com.google.gson.annotations.SerializedName
+import java.io.Serializable
+
+/**
+ * A simple class to capture how most people will express a waypoint.
+ */
+data class SimpleWaypoint (
```

Comment:
Why couldn't this be called `Waypoint`?

Reply:
Merged Waypoint and SimpleWaypoint into Waypoint